### ISSUE TITLE
Common technical words

### ISSUE THREAD

**Issue by aplanas** [thecodix/book#2]
I guess that we are used to read in English, and somehow we have an understanding of some words that, one are expressed in Spanish, sounds 'wrong'. For example 'ownership', we can use 'propiedad', 'pertenencia', 'posesion', so even if no one sounds good, we need to agree in those terms to provide some coherence.

I suggest to use a Wiki entry to have a table to reflect such agreements.

**Comment by thecodix** [comment]
Hello Alberto (hope you don't mind I use your name), a wiki table sounds useful to have a reference and agree on some standards.

Feel free to start one, and we will keep adding all terms that can be controversial to it with corresponding agreed term :)

**Comment by goyox86** [comment]
Yeah that would be useful @aplanas and then we can reference it in the README and CONTRIBUTING. Either way we can start discussing here! Right? If so, I think the keys concepts to be translated are:

1. - Ownership = Posesion (Originally I used "Pertenecia" in my first translation of the book but I've come to like this one more.
2. - Borrowing = Prestamo.
3. - Lifetime = Tiempo de Vida (I never been happy with this one but that was the best I could find in my original translation).
4. - Memory safety = Seguridad en el manejo de memoria (Again not happy with this one) .
5. - Data race = Condicion de carrera.
6. - Concurrency = Concurrencia.

Just out of the top of my head, tho

**Comment by thecodix** [comment]
I opened a wiki page for it
https://github.com/thecodix/book/wiki/Translation-of-complicated-words

Please keep adding the ones, and put comments next to the other where you don't completely agree
